env.js

```diff
@@ -1,22 +1,22 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
-export const { DB_NAME, MONGO_PASSWORD, API_KEY, SALT_ROUNDS } = process.env;
+export const { MONGO_URI, API_KEY, SALT_ROUNDS } = process.env;
 
 export const API_PORT = process.env.API_PORT || 5000;
 
-export let DB_URL = '';
+export let DB_URL;
 export let hosts = [];
 export let callbackUrl = '';
 export const TURN_INTERVAL = 0;
 if (process.env.NODE_ENV === 'production') {
     // React App Deployed URLs
     hosts = ['https://bundly.tech', 'https://dash-delta.vercel.app', 'https://dash.bundly.tech'];
     callbackUrl = 'https://bundly.tech/api';
-    DB_URL = 'mongodb://mongo:27017/dash'
-  } else {
+    DB_URL = 'mongodb://localhost:27017/dash';
```

Comment:
```suggestion
    DB_URL = 'mongodb://mongo:27017/dash';
```

We using a docker image for running mongo, hence the URL